src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -123,15 +132,17 @@
     }
 
     /**
-     * Returns a {@code Set} of non-binary files for the repo {@code repoConfig}.
+     * Returns a {@code Set} of non-binary files for the repo {@code repoConfig}
+     * if {@code isBinaryFiles} is set to `false`.
+     * Otherwise, returns a {@code Set} of binary files for the repo {@code repoConfig}
      */
-    public static Set<Path> getNonBinaryFilesList(RepoConfiguration repoConfig) {
+    public static Set<Path> getFilesList(RepoConfiguration repoConfig, boolean isBinaryFile) {
         List<String> modifiedFileList = GitDiff.getModifiedFilesList(Paths.get(repoConfig.getRepoRoot()));
 
-        // Gets rid of binary files and files with invalid directory name.
```

Comment:
i think the original comment is simpler, should keep it

Reply:
But this function can either get rid of binary files, or keep only binary files. The original comment would be misleading.

Reply:
i see then maybe reword the sentence, `filters by the {@code isBinaryFile} flag.` will be better

Reply:
i see then maybe reword the sentence, `filters by the {@code isBinaryFile} flag.` will be better

Reply:
Ok changed.